### PR TITLE
WIP: Add separate docs for Display Rules

### PR DESCRIPTION
**Pull request by markhougaard**

### CODE REVIEW

File: src/various/display-rules.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    order: 500
+---
+
+## What are Display Rules
+
+In this guide you will be introduced to the concept of Display Rules and learn how you can use Display Rules to change how Locations (POIs, Rooms and Areas) are displayed on the map.
+
+It is possible to apply Display Rules to a single Location using its ID, or to all Locations of a specific Location Type such as `MEETING_ROOMS`.
+
+The MapsIndoors CMS is used to control the default appearance of Types and Locations. To change the default appearance at _runtime_ you can use Display Rules.
```
Comment (kasperlaurberg) [inline src/various/display-rules.md:16]: They are also used to change the default appearance, but that might be to much info

File: src/various/display-rules.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+It is possible to control different aspects of the appearance. For example at what zoom levels Labels and Icons should be visible.
+
+## Display Rule Hierarchy
+
+In each MapsIndoors SDK, a "Main Display Rule" outlines a list of sensible defaults for all geodata. Each Location Type inherits its values from this Main Display Rule, unless specifically overriden for that Type. Each Location (Room, Area or POI) in turn inherits the combined values from the Main Display Rule and its Location Type, unless for those properties where it's been specifically overriden on that Location.
+
+As an example, you might want all polygons to be red. However, the Location Type for "Meeting Room" specifies that their polygons should be blue, while the "Executive Meeting Room" Location specifically has an orange polygon.
+
+No matter what is specified in this hierarchy, you can override it _runtime_ in your app. That means all matches for a specific search query can be specified to have a polygon color that is red, regardless of what exisits in the defaults, on their Types and for those Locations specifically.
```
Comment (kasperlaurberg) [inline src/various/display-rules.md:26]: > That means all matches for a specific search query can be specified to have a polygon color that is red, regardless of what exisits in the defaults, on their Types and for those Locations specifically

Present it as an example. Comes off as the only option

**Review by kasperlaurberg — APPROVED** [review]

File: src/various/display-rules.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    parent: various
+    title: Display Rules
+    order: 500
+---
+
+## What are Display Rules
+
+In this guide you will be introduced to the concept of Display Rules and how you can use Display Rules to change how Locations (POIs, Rooms and Areas) are displayed on the map.
+
+The MapsIndoors CMS is used to control the default appearance of Types and Locations. To change the default appearance at _runtime_ you can use Display Rules.
```
Comment (WJesper) [inline src/various/display-rules.md:14]: Perhaps remove "Types" as it becomes a bit complicated to explain. 

Perhaps: The MapsIndoors CMS is used to control the initial appearance Locations. However, the Display Rules and thereby the appearance of Locations can be changed from the application at _runtime_ you, giving the application the power to change Locations based on events in the app.

File: src/various/display-rules.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+## Display Rule Hierarchy
+
+In each MapsIndoors SDK, a "Main Display Rule" outlines a list of sensible defaults for all geodata. Each Location Type inherits its values from this Main Display Rule, unless specifically overriden for that Type. Each Location (Room, Area or POI) in turn inherits the combined values from the Main Display Rule and its Location Type, unless for those properties where it's been specifically overriden on that Location.
```
Comment (WJesper) [inline src/various/display-rules.md:18]: Change:  " unless specifically overriden for that Type"  to " unless the value us specifically overriden" 
Change: "Each Location (Room, Area or POI) in turn inherits the combined values from the Main Display Rule and its Location Type" to "Each Location (Room, Area or POI) uses the Display Rule from the Type" excepts for values that are specifically set for the individual Location.

File: src/various/display-rules.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+As an example, you might want all polygons to be `red`. However, the Location Type for "Meeting Room" specifies that their polygons should be `blue`, while the "Executive Meeting Room" Location specifically has an `orange` polygon.
```
Comment (WJesper) [inline src/various/display-rules.md:20]: I would show the figure and explain it in detail before doing the example..

File: src/various/display-rules.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+All properties are optional.
+
+| Properties | Type | Validation | Description |
+| --- | --- | --- | --- |
+| visible | boolean | | Must be true for the location to be shown in the map. If not true all other parameters are ignored. |
+| iconVisible | boolean | | Must be `true` for the icon to be visible in the map. If this is not `true` the `zoomFrom`, `zoomTo`, `iconUrl`, `iconScale` and `iconSize` parameters are ignored. |
+| zoomFrom | number | `1`-`21` (`22` if applicable) | The minimum zoom level the image/icon will be visible on the map. |
+| zoomTo | number | `1`-`21` (`22` if applicable) | The maximum zoom level the image/icon will be visible on the map. |
+| icon | string  | | A URL to an image to represent the Location on the map. |
+| iconScale | double | `> 0` | ⚠️ Deprecated |
```
Comment (JG1995) [inline src/various/display-rules.md:78]: If it's deprecated shouldn't it just be removed from the table?

**Review by JG1995 — APPROVED** [review]
Made a single comment, but feel free to leave it in if it's important.

**Pull request merged**